dbms/src/Flash/tests/gtest_aggregation_executor.cpp

```diff
@@ -282,17 +284,27 @@ try
     std::shared_ptr<tipb::DAGRequest> request;
     auto agg_func0 = Count(col(col_name[0])); /// select count(age) from clerk group by country;
     auto agg_func1 = Count(col(col_name[1])); /// select count(gender) from clerk group by country, gender;
-    std::vector<MockAstVec> agg_funcs = {{agg_func0}, {agg_func1}};
+    auto agg_func2 = Count(lit(Field(static_cast<UInt64>(1)))); /// select count(1) from clerk;
+    auto agg_func3 = Count(lit(Field())); /// select count(NULL) from clerk;
+    auto agg_func4 = Count(lit(Field(static_cast<UInt64>(1)))); /// select count(1) from clerk group by country;
+    auto agg_func5 = Count(lit(Field())); /// select count(NULL) from clerk group by country;
```

Comment:
and add unit tests for count(not null column)?

Reply:
Sorry, Is `count(not null column)` already in the original ut?

Reply:
Currently UT framework cannot create nullable columns in mock table.
We can consider testing `count(nullable column)` in fullstack-test

Reply:
Currently UT framework cannot create not null columns in mock table.
We can consider testing count(not null column) in fullstack-test

Reply:
> Currently UT framework cannot create nullable columns in mock table. We can consider testing `count(nullable column)` in fullstack-test

typo

Reply:
there are nullable column in ut before. I have added `count(not null column)` case. PTAL

Reply:
There is no way to specify whether the column in MockTable is not null, so the column `pr` is also nullable.
This is a limitation of the current UT framework...

Reply:
> There is no way to specify whether the column in MockTable is not null, so the column `pr` is also nullable.
> This is a limitation of the current UT framework...

ok...... will supply tests later.